R/optionalInput.R

```diff
@@ -109,6 +135,7 @@ optionalSelectInput <- function(inputId, # nolint
     checkmate::check_class(label_help, "html")
   )
   checkmate::assert_flag(fixed)
+  singleton(shinyjs::useShinyjs())
```

Comment:
is `singleton` necessary?

Reply:
Probably not, `shinyjs` should have it already. I'll remove `singleton` call